compiler/src/iree/compiler/Dialect/Stream/IR/test/channel_ops.mlir

```diff
@@ -0,0 +1,38 @@
+// RUN: iree-opt --split-input-file %s | iree-opt --split-input-file | FileCheck %s
```

Comment:
Hmm, I wonder why there are two iree-opt? If there is a need, then it might be good to have a comment for this.

Reply:
this ensures that we can round-trip the IR (it's parsed and printed by the first iree-opt and then parsed again by the second) - it's a convention we have in the IR/test/* files of various dialects